src/librustc_borrowck/diagnostics.rs

```diff
@@ -286,6 +286,54 @@ You can read more about cell types in the API documentation:
 https://doc.rust-lang.org/std/cell/
 "##,
 
+E0389: r##"
+An attempt was made to assign to data in a non-mutable reference. Example:
```

Comment:
Ah, okay. The RFC doesn't seem particularly consistent-- it uses "Example of erroneous code:", "Erroneous code example:", and other errors currently in the same file as E0389 & co. use "For example:", and "Example:". It makes sense to have a standard, though, so perhaps there should be a PR changing the old ones. I can take a look at that after this one if you'd like.